src/app.tsx

```diff
@@ -100,8 +162,65 @@ function App() {
       <Button variant="dark" onClick={dummy}>
         Hello
       </Button>
+
+      <div className="flashcards-container">
+        <Flashcard card={readonlyCard} variant={'readonly'} />
+        <Reorder.Group axis="y" values={cards} onReorder={setCards}>
+          {getFlashcards()}
+        </Reorder.Group>
+      </div>
     </div>
   );
+
+  function getFlashcards() {
+    return cards.map((card, id) => {
+      const index = id + 1;
+      const value = (
+        <Flashcard
+          key={index}
+          card={card}
+          index={index}
+          variant={activeCard === card.id ? 'active' : 'inactive'}
+          onFocus={() => setActiveCard(card.id)}
+          onCardChange={(c) => handleCardChange(c, id)}
+          onDownClick={() => handleDownClick(id)}
+          onUpClick={() => handleUpClick(id)}
+          onRemoveClick={() => handleRemoveClick(id)}
+        />
+      );
+      return (
+        <Reorder.Item key={card.id} value={card}>
+          {value}
```

Comment:
nit: Would it better to get rid of the variable and put the jsx here? I found myself looking up and down

also framer motion is so cool 😮

Reply:
Yeah, it's the same kind of makeshift parent component again, haha. I'll have it fixed with the new one